### ISSUE TITLE
[Bug]: Focusing on Skeleton in narriator sometimes does not announce the aria-label (Even the default one)

### ISSUE THREAD

**Issue by snisonoff-microsoft-unmanaged** [microsoft/fluentui#31194]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
Repro steps:

-Open linked code sandbox
-Open narrator
-navigate to "Open new tab" button
-press down button
-hear incorrect announcement when focused on Skeleton
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/p/sandbox/skeleton-m96nz6

## Actual Behavior
Announces irrelevant information such as the page name

## Expected Behavior
should announce default aria-label "loading content"

### Requested priority

Blocking

### Products/sites affected

TwoUI

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by dmytrokirpa** [comment]
Thank you, @snisonoff-microsoft-unmanaged, for reporting the issue. I have tested it several times and am unable to reproduce it now. @snisonoff-microsoft-unmanaged, could you please confirm whether the issue still occurs for you, or if there are any preconditions or specific steps to reproduce it beyond those mentioned in the description?
